samza-core/src/main/java/org/apache/samza/zk/ZkUtils.java

```diff
@@ -190,7 +188,7 @@ public void publishJobModel(String jobModelVersion, JobModel jobModel) {
    * @return job model for this version
    */
   public JobModel getJobModel(String jobModelVersion) {
-    LOG.info("pid=" + processorId + "read the model ver=" + jobModelVersion + " from " + keyBuilder.getJobModelPath(jobModelVersion));
+    LOG.info("read the model ver=" + jobModelVersion + " from " + keyBuilder.getJobModelPath(jobModelVersion));
```

Comment:
Yeah. I am going to change this. I think I already responded to @nickpan47 's comments before.